http/src/request/guild/user/update_current_user_voice_state.rs

```diff
@@ -0,0 +1,84 @@
+use crate::request::prelude::*;
+use twilight_model::id::{ChannelId, GuildId};
+
+#[derive(Serialize)]
+struct UpdateCurrentUserVoiceStateFields {
+    channel_id: ChannelId,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    suppress: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    request_to_speak_timestamp: Option<String>,
```

Comment:
This needs to be changed to a `Option<Option<String>>` so it is possible to both have the case where it is not there and the case where it is set to `null` as both are possible cf. the docs:

| Field                       | Type               | Description                                    |
| --------------------------- | ------------------ | ---------------------------------------------- |
| request_to_speak_timestamp? | ?ISO8601 timestamp | sets the user's request to speak               |